backend/API/API/HostedServices/MessageService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.Hosting;
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace API.HostedServices
+{
+    public class MessageService : BackgroundService
+    {
+        IHubContext<Hubs.MessageHub> hubContext;
+
+        public MessageService(IHubContext<Hubs.MessageHub> hubContext)
+        {
+            this.hubContext = hubContext;
+        }
+        
+
+        protected async override Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                await Task.Delay(5000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4610207)